### DISCUSSION TITLE
Low Detail at Hallowed Sepulchre

### DISCUSSION THREAD

**Discussion by EMB222** [runelite/runelite#18250] [Q&A] [answered]
The Low Detail plugin used to remove all of the floor textures in sepulchre but as of a few weeks ago, the textures on the first floor are patchy and none of the textures on the rest of the floors are removed.

The image on the left side is from a recording a little while ago and the one on the right is my client right now.

![sep](https://github.com/user-attachments/assets/dc09c18e-fce5-473f-a7cc-f757812e33e0)

Any help would be appreciated :)

**Comment by Felanbird** [comment, marked as answer]
Jagex made a change so low detail no longer works on most Sepulchre floors, and in Verzik's room.

The available workaround is using `Ground Object Hider` from the hub.

**Reply by EMB222** [reply]
Ah that is very frustrating, thank you for your quick response.
